musicbot/bot.py

```diff
@@ -939,6 +939,11 @@ def run(self):
             for s in self.guilds:
                 ser = ('{} (unavailable)'.format(s.name) if s.unavailable else s.name)
                 log.info(' - ' + ser)
+                if self.config.leavenonowners:
+                    check = s.get_member(owner.id)
```

Comment:
This may have an unintended result if the server is unavailable.

Reply:
I'll add in a try and except?

Reply:
I have a suspicion that if a server is unavailable and you try to get a member, it'll return None regardless of if the member is in there or not because the data may not be obtainable. I'd rather err on the side of caution here and check to make sure that the server is available first.

Reply:
Makes sense, I'll add in
```If s.unavailable:
    unavailable_servers += 1
    log.info('Not going through with a check in {} servers due to unavailability.'format(unavailable_servers))```
As well as a counter right below `if self.config.leavenonowners`